### PR TITLE
Remove a broken link in Readme.md

### PR DESCRIPTION
**Pull request by zaqim**
## Description
The link for open file limit was dead. Instead of putting up a link, I've added a solution in a short and compact way. I've also added the solution for docker since docker does not allow ulimit -n command unless it is a privileged mode (which is not suggested to use).

### CODE REVIEW

**Review by patkivikram — APPROVED** [review]

**Comment by zaqim** [comment]
I did not change any coding part but the tests seem to give this error:

> faust/streams.py:397:89: E501 line too long (91 > 88 characters)

**Comment by wbarnha** [comment]
If you run `scripts/lint`, the errors should go away

**Comment by wbarnha** [comment]
> Files changed: 116

This just got much weirder. I'll see to including your commit in https://github.com/faust-streaming/faust/pull/332/commits/e6c989207445db5049ccdb2569f92509a17233f6 somehow if the checks don't work.

File: faust/agents/agent.py [hunk 1]
```diff
@@ -312,9 +310,7 @@ async def _on_start_supervisor(self) -> None:
         channel: ChannelT = cast(ChannelT, None)
         for i in range(self.concurrency):
             res = await self._start_one(
-                index=i,
-                active_partitions=active_partitions,
-                channel=channel,
+                index=i, active_partitions=active_partitions, channel=channel,
```
Comment (wbarnha) [inline faust/agents/agent.py:313]: That's unusual, it's like it did the _opposite_ of being linted.

**Comment by zaqim** [comment]
I ran it but to many files are changed, it might create merge 

> > Files changed: 116
> 
> This just got much weirder. I'll see to including your commit in [e6c9892](https://github.com/faust-streaming/faust/commit/e6c989207445db5049ccdb2569f92509a17233f6) somehow if the checks don't work.

I  agree, 116 file changes will probably create merge conflicts

**Comment by wbarnha** [comment]
I ran this after checking out your branch:
```
REDACTED@REDACTED:~/PycharmProjects/faust$ ./scripts/lint 
+ autoflake --in-place --recursive faust tests setup.py
+ isort --project=faust faust tests setup.py
+ black faust tests setup.py
reformatted faust/auth.py
reformatted faust/contrib/sentry.py
reformatted faust/cli/worker.py
reformatted faust/livecheck/models.py
reformatted faust/livecheck/signals.py
reformatted faust/assignor/client_assignment.py
reformatted faust/agents/manager.py
reformatted faust/__init__.py
reformatted faust/assignor/copartitioned_assignor.py
reformatted faust/livecheck/runners.py
reformatted faust/livecheck/app.py
reformatted faust/models/base.py
reformatted faust/serializers/schemas.py
reformatted faust/assignor/partition_assignor.py
reformatted faust/channels.py
reformatted faust/sensors/statsd.py
reformatted faust/tables/objects.py
reformatted faust/tables/manager.py
reformatted faust/models/record.py
reformatted faust/models/fields.py
reformatted faust/sensors/datadog.py
reformatted faust/tables/table.py
reformatted faust/sensors/prometheus.py
reformatted faust/transport/base.py
reformatted faust/stores/rocksdb.py
reformatted faust/models/typing.py
reformatted faust/sensors/monitor.py
reformatted faust/tables/sets.py
reformatted faust/types/settings/base.py
reformatted faust/types/stores.py
reformatted faust/types/models.py
reformatted faust/types/settings/sections.py
reformatted faust/topics.py
reformatted faust/types/agents.py
reformatted faust/tables/base.py
reformatted faust/utils/platforms.py
reformatted faust/utils/json.py
reformatted faust/types/web.py
reformatted tests/bench/forward.py
reformatted faust/agents/agent.py
reformatted faust/types/app.py
reformatted faust/utils/codegen.py
reformatted tests/functional/agents/test_concurrency.py
reformatted setup.py
reformatted faust/web/views.py
reformatted faust/streams.py
reformatted faust/web/drivers/aiohttp.py
reformatted faust/types/tables.py
reformatted tests/functional/agents/test_isolated_partitions.py
reformatted tests/consistency/test_consistency.py
reformatted faust/web/base.py
reformatted tests/functional/conftest.py
reformatted faust/types/settings/params.py
reformatted tests/regression/i323/proj323/faust.py
reformatted tests/regression/i324/proj324/faust.py
reformatted tests/conftest.py
reformatted tests/functional/web/test_views.py
reformatted tests/functional/serializers/test_registry.py
reformatted tests/stress/app.py
reformatted faust/tables/recovery.py
reformatted tests/stress/reports/app.py
reformatted tests/stress/tests/simple/app.py
reformatted tests/stress/reports/collector.py
reformatted tests/stress/tests/forwarder/app.py
reformatted tests/meticulous/assignor/test_copartitioned_assignor.py
reformatted tests/stress/tests/tables/app.py
reformatted tests/functional/test_channels.py
reformatted tests/unit/agents/test_manager.py
reformatted tests/unit/cli/test_clean_versions.py
reformatted tests/unit/cli/test_params.py
reformatted tests/unit/conftest.py
reformatted tests/unit/app/test_router.py
reformatted tests/unit/app/test_service.py
reformatted faust/app/base.py
reformatted tests/functional/models/test_typing.py
reformatted tests/unit/agents/test_replies.py
reformatted faust/transport/consumer.py
reformatted faust/types/settings/settings.py
reformatted tests/unit/models/test_fields.py
reformatted tests/functional/web/test_cache.py
reformatted tests/unit/livecheck/test_app.py
reformatted tests/unit/livecheck/test_runners.py
reformatted tests/unit/tables/test_objects.py
reformatted tests/unit/stores/test_base.py
reformatted tests/unit/stores/test_aerospike.py
reformatted tests/unit/sensors/test_prometheus.py
reformatted tests/unit/sensors/test_statsd.py
reformatted tests/unit/sensors/test_monitor.py
reformatted tests/unit/sensors/test_datadog.py
reformatted tests/unit/tables/test_table.py
reformatted tests/unit/livecheck/test_case.py
reformatted tests/unit/cli/test_base.py
reformatted tests/unit/test_streams.py
reformatted tests/unit/test_worker.py
reformatted tests/unit/tables/test_recovery.py
reformatted tests/unit/agents/test_agent.py
reformatted tests/unit/transport/test_conductor.py
reformatted tests/unit/tables/test_base.py
reformatted tests/unit/utils/test_iso8601.py
reformatted tests/unit/web/test_blueprints.py
reformatted tests/unit/tables/test_sets.py
reformatted tests/unit/utils/terminal/test_tables.py
reformatted tests/unit/stores/test_rocksdb.py
reformatted tests/unit/utils/terminal/test_spinners.py
reformatted tests/unit/tables/test_wrappers.py
reformatted tests/unit/utils/test_urls.py
reformatted tests/unit/test_topics.py
reformatted faust/transport/drivers/aiokafka.py
reformatted tests/functional/test_app.py
reformatted tests/unit/transport/test_producer.py
reformatted tests/unit/app/test_base.py
reformatted tests/unit/web/drivers/test_aiohttp.py
reformatted tests/functional/test_models.py
reformatted tests/unit/transport/test_consumer.py
reformatted tests/unit/transport/drivers/test_aiokafka.py

All done! ✨ 🍰 ✨
115 files reformatted, 227 files left unchanged.
```

**Comment by zaqim** [comment]
I get: 
> + autoflake --in-place --recursive faust tests setup.py
> + isort --project=faust faust tests setup.py
> + black faust tests setup.py
> All done! ✨ 🍰 ✨
> 342 files left unchanged.

Should I configure autoflake?

**Comment by wbarnha** [comment]
> I get:
> 
> > * autoflake --in-place --recursive faust tests setup.py
> > * isort --project=faust faust tests setup.py
> > * black faust tests setup.py
> >   All done! sparkles cake sparkles
> >   342 files left unchanged.
> 
> Should I configure autoflake?

What is your current install of autoflake? Try running `pip3 install autoflake --upgrade` and lint again?

**Comment by wbarnha** [comment]
I forgot, try `pip3 install black --upgrade`, then linting should work right

**Comment by wbarnha** [comment]
Fixed it by manually editing the code. This issue always comes up when `flake8` or some other package used in `./scripts/check` gets unexpectedly updated.

**Comment by zaqim** [comment]
> Fixed it by manually editing the code. This issue always comes up when `flake8` or some other package used in `./scripts/check` gets unexpectedly updated. I apologize for the inconvenience.

No problem :) thanks

**Pull request merged**